package.json

```diff
@@ -50,7 +50,7 @@
   },
   "dependencies": {
     "axios": "^0.15.3",
-    "chart.js": "^2.5.0",
+    "chart.js": "chartjs/Chart.js",
```

Comment:
Use chart.js master for now until server rendering is released on npm: https://github.com/chartjs/Chart.js/pull/3909 (merged into master, but not yet released on npm)